test/problems/identity/generators/generators.yaml

```diff
@@ -89,36 +89,36 @@ data:
       zero_prefix_group:
         data:
           # Both ~ (None) and the empty string can be used to have only numbers as testcase names.
-          - ~: stdout.py 123 # 01.in
-          - "": stdout.py 124
-          - "": stdout.py 125
-          - "": stdout.py 126
-          - "": stdout.py 127
-          - "": stdout.py 128
-          - "": stdout.py 129
-          - "": stdout.py 130
-          - "": stdout.py 131
-          - "": stdout.py 132 # 10.in
+          - null: stdout.py 123 # 01.in
```

Comment:
Funky, does a literal `null` actually come out as `None` in Python? :joy: Perhaps better to keep using `~` in this example, that's more explicit. I don't think we need any special handling in `bt upgrade` for this though

Reply:
no idea, but wil fix :D